thoth/storages/graph/postgres.py

```diff
@@ -3606,6 +3607,54 @@ def _runtime_environment_conf2models(
 
         return hardware_information, software_environment
 
+    def create_github_app_installation(self, install_data: List[dict]) -> int:
```

Comment:
Can the method accept parameters directly (not via a dictionary) so the calling protocol is enforced? The return value can be a boolean that states if the given installation already exists.

The iteration for creating multiple installations will be left up to callee to ensure correct number of installations is created.

Reply:
Yes, let me change that. Thanks!